cocos/serialization/deserialize-dynamic.ts

```diff
@@ -136,17 +136,8 @@ function compileDeserializeJIT (self: _Deserializer, klass: CCClassConstructor<u
         const defaultValue = CCClass.getDefault(attrs[propName + POSTFIX_DEFAULT]);
         const userType = attrs[propName + POSTFIX_TYPE] as AnyFunction | string | undefined;
         if (fastMode && (defaultValue !== undefined || userType)) {
-            let isPrimitiveType;
-            if (defaultValue === undefined) {
-                isPrimitiveType = userType instanceof CCClass.Attr.PrimitiveType || userType === ENUM_TAG || userType === BITMASK_TAG;
-            } else {
-                const defaultType = typeof defaultValue;
-                isPrimitiveType = defaultType === 'string'
-                                  || defaultType === 'number'
-                                  || defaultType === 'boolean';
-            }
-
-            if (isPrimitiveType) {
+            const isPrimitiveTypeInFastMode = checkIsPrimitiveTypeInFastMode(defaultValue, userType);
```

Comment:
Can i know why name it as fast mode?

Reply:
You can ask @jareguo

Reply:
Yes, just as shrinktofit's remarks. FastMode does not verify every instance data that is loaded, but simply processes it uniformly. This is an optimization strategy that can only be used when the types of instance data are certain to match, and is currently only effective for types built into the engine. When the type of the engine really changes and needs to be compatible, the editor can complete the data migration, so all types of the engine are fast. However, the instance data in the user project may change continuously as the code is adjusted, so the loaded data needs to be checked twice.

Reply:
Good to know. Thanks.